### PR TITLE
License change

### PR DESCRIPTION
**Pull request by calevans**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 NOCWorx L.L.C.
```
Comment (adrian-enspired) [inline LICENSE]: is spelled "NocWorx".  Need a codesniff for this.

Comment (calevans) [reply LICENSE]: Copied that straight out of the docblock at the head of every file.

Comment (hassan-bazzi) [reply LICENSE]: The comment block that's in every PHP file spells it NOCWorx L.L.C.

Comment (adrian-enspired) [reply LICENSE]: i know (not blaming you). that's old, new stuff should be `NocWorx`.

and the "codesniff" part was mainly a joke  :)

Comment (calevans) [reply LICENSE]: done

**Comment by robertology** [comment]
@clwells change to MIT license officially approved?

**Comment by clwells** [comment]
+1

**Pull request merged**